reporters_db/data/laws.json

```diff
@@ -4817,6 +4885,23 @@
             "variations": []
         }
     ],
+    "Pan-Am. T.S.": [
+        {
+            "cite_type": "treaty",
+            "end": null,
+            "examples": [
+                "Pan-Am. T.S. No. 22"
+            ],
+            "href": "https://law.resource.org/pub/us/code/blue/IndigoBook.html#T6",
+            "jurisdiction": "",
+            "name": "Pan-American Treaty Series",
+            "regexes": [
+                "$reporter\\s*(?:No\\.|Number)?\\s*(?P<number>\\d+)"
```

Comment:
This is a minor quibble, but should we add number/No ?pnumber to a regex pattern if we are going to be using it this much?

Reply:
Sure. I am happy to do that.

Reply:
I'm not sure if its right or not.  It was just ... there a bunch. Probably doesnt hurt